projects/1.20/assets/somanyapples/so_many_apples/lang/zh_cn.json

```
@@ -0,0 +1,35 @@
+{
+  "item.so_many_apples.end_apple": "末地苹果",
+  "item.so_many_apples.netherite_apple": "下界合金苹果",
+  "item.so_many_apples.nether_star_apple": "下界之星苹果",
+  "item.so_many_apples.nether_apple": "下界苹果",
+  "item.so_many_apples.night_apple": "黑夜苹果",
+  "item.so_many_apples.tnt_apple": "TNT苹果",
+  "item.so_many_apples.water_apple": "海洋苹果",
+  "item.so_many_apples.bedrock_apple": "基岩苹果",
+  "item.so_many_apples.copper_apple": "铜苹果",
+  "item.so_many_apples.experience_apple": "经验苹果",
+  "item.so_many_apples.sacrifice_apple": "献祭苹果",
+  "item.so_many_apples.diamond_apple": "钻石苹果",
+  "item.so_many_apples.green_apple": "青苹果",
+  "item_group.so_many_apples.so_many_apples_tab": "超多苹果",
+  "item.so_many_apples.day_apple": "白昼苹果",
+  "item.so_many_apples.slime_apple": "史莱姆苹果",
+  "item.so_many_apples.golden_carrot_apple": "金胡萝卜苹果",
+  "item.so_many_apples.glass_apple": "玻璃苹果",
+  "item.so_many_apples.villager_apple": "村民苹果",
+  "item.so_many_apples.glowstone_apple": "荧石苹果",
+  "item.so_many_apples.caramelized_apple": "蜂蜜苹果",
+  "item.so_many_apples.command_apple": "指令苹果",
```

```suggestion
  "item.so_many_apples.command_apple": "命令苹果",
```
https://zh.minecraft.wiki/w/%E5%91%BD%E4%BB%A4